src/frontend/components/UI/TextInputField/index.tsx

```diff
@@ -35,7 +35,7 @@ const TextInputField = ({
       {label && <label htmlFor={htmlId}>{label}</label>}
       {inputIcon}
       <input type="text" id={htmlId} value={value} {...inputProps} />
-      {value && warning}
+      {warning}
```

Comment:
I was nervous about the blast radius of changing this logic in the main `TextInputField` component but it actually looks like the `warning` prop is not currently being used anywhere in the app. The way I see it, the `warning` shouldn't hinge on the existence of a `value`, but if people disagree with me on that, I think I can just use `afterInput` to display the empty path warning.

Reply:
Yup this is fine. If the component using this has any specific preferences on when `warning` should be displayed, it can handle those itself